app/src/main/res/layout/activity_main.xml

```diff
@@ -66,7 +66,19 @@
                 android:paddingTop="@dimen/activity_margin_xsmall"
                 android:paddingBottom="@dimen/fab_recyclerview_padding"
                 android:clipToPadding="false" />
-
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:paddingStart="@dimen/activity_margin_small"
+                android:paddingEnd="@dimen/activity_margin_small"
```

Comment:
I would prefer `activity_margin_large` here, it just looks better in my opinion.

Reply:
Changed it